platforms/ios/src/TangramMap/TGMarker.mm

```diff
@@ -57,13 +57,24 @@ - (instancetype)initWithMapView:(TGMapViewController *)mapView
     return self;
 }
 
-- (void)setStyling:(NSString *)styling
+- (void)setStylingString:(NSString *)styling
```

Comment:
Ahh I see. I did not look at the `@properties` at all.

Anyways I think it would make sense to have them named like what you suggested above.